tests/script-based-pre/build-cache-bin/cache_works.expected

```diff
@@ -0,0 +1,25 @@
+Initial compilation
+target/initial.log:1
```

Comment:
No idea where this number is printed from 😄

Reply:
hahahaha.... it is printed from the "grep -c Compiling". I will improve the message, but it is printing the number of crates that were compiled. :)